src/components/FlowsDeleteButton.vue

```diff
@@ -1,7 +1,5 @@
 <template>
-  <Transition name="flows-delete-button-transition">
-    <p-button v-if="selected.length > 0" danger icon="TrashIcon" @click="open" />
-  </Transition>
+  <p-button v-if="selected.length > 0" v-bind="attrs" danger icon="TrashIcon" @click="open" />
```

Comment:
This transition was a bit of an anomaly compared to similar functioning buttons - not against transitions overall but it felt superfluous

Reply:
Totally agree